scripts/qa_crowbarsetup.sh

```diff
@@ -675,7 +675,7 @@ function addcloud6pool()
 function addcctdepsrepo()
 {
     if [[ "$cloudsource" == @(develcloud5|GM5|GM5+up) ]]; then
-        zypper ar -f http://$susedownload/ibs/Devel:/Cloud:/Shared:/Rubygem/SLE_11_SP3/Devel:Cloud:Shared:Rubygem.repo
+        zypper --non-interactive --no-gpg-checks ar -f http://$susedownload/ibs/Devel:/Cloud:/Shared:/Rubygem/SLE_11_SP3/Devel:Cloud:Shared:Rubygem.repo
```

Comment:
It was an attempt to fix a local test failure which dint work anyways. Will remove in the next update to the PR. Thanks